### DISCUSSION TITLE
anyway to remove realtionmanager class names from url?

### DISCUSSION THREAD

**Discussion by ehs4nm** [filamentphp/filament#1724] [Help]
Hi.
adding a relation manager to a resource adds resource relation class to the page URL . something like this:
"http://127.0.0.1:8000/services/therapy_with/edit?activeRelationManager=App%5CFilament%5CResources%5CServiceResource%5CRelationManagers%5COrdersRelationManager"
is there any workaround to hide this from clients?

**Comment by danharrin** [comment]
I was talking about this with someone else the other day, we should define a static `getSlug()` method for relation managers where you can customise this.

**Reply by Z3d0X** [reply]
How about like this in the `Resource`,  might be easier to implement?

```
public static function getRelations(): array
{
    return [
        'permissions' => RelationManagers\PermissionsRelationManager::class,
        'users' => RelationManagers\UsersRelationManager::class,
    ];
}
```

**Reply by danharrin** [reply]
Thats a very good idea!
